pilot/pkg/bootstrap/server.go

```diff
@@ -254,10 +254,15 @@ func NewServer(args *PilotArgs) (*Server, error) {
 	if err := s.initMonitor(args.DiscoveryOptions.MonitoringAddr); err != nil {
 		return nil, fmt.Errorf("error initializing monitor: %v", err)
 	}
+	args.Config.ControllerOptions.CAROOT = ""
+	if features.CentralIstioD {
+		if s.ca != nil && s.ca.GetCAKeyCertBundle() != nil {
+			args.Config.ControllerOptions.CAROOT = string(s.ca.GetCAKeyCertBundle().GetRootCertPem())
```

Comment:
nit: if you want to get root cert rotation for "free" just change CAROOT to be `func() string` instead. Or `func() map[string]string` which is the input to the namespace controller. That is what the local one does